sql/moz-fx-data-shared-prod/google_ads_derived/android_app_campaign_stats_v1/query.sql

```diff
@@ -48,6 +48,32 @@ retention_aggs AS (
     date,
     campaign_id,
     ad_group_id
+),
+by_ad_group_id AS (
+  SELECT
+    date,
```

Comment:
```suggestion
    `date`,
```

I believe `date` is a keyword, perhaps it would be best to surround it with back-ticks to make sure it's treated as a string literal?

Reply:
hm, the formatter knows it's an identifier and not the built-in function, or else it would have capitalized it